invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js

```diff
@@ -6,44 +6,82 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-import React from "react";
-import { Input } from "semantic-ui-react";
-import { i18next } from "@translations/invenio_communities/i18next";
+import React, { Component } from "react";
+import { i18next } from "@translations/invenio_search_ui/i18next";
```

Comment:
Just make sure this gets updated when https://github.com/inveniosoftware/invenio-administration/issues/17 is merged :)